ansible/environments.yml

```diff
@@ -103,3 +103,18 @@
           image_name: "d324da4b-b827-44fb-8331-bfad41a28aca"
       tags:
         - dev2-env
+
+    - name: Ensure aws_dev environment stack
+      cloudformation:
+        stack_name: "ae-aws-dev"
+        state: "present"
+        region: "us-west-1"
+        disable_rollback: true
+        template: "../aws/ae-environment.yml"
+      args:
+        template_parameters:
+          envname: "dev_aws"
+          apiport: 3013
+          nodecount: 10
+          amiid: "ami-8c076ce3"
```

Comment:
This has a nice comment in the past about the image content (ubuntu+master user .. etc.)

Reply:
added